sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/models/EvaluationEvent.java

```diff
@@ -0,0 +1,86 @@
+package com.azure.spring.cloud.feature.management.models;
+
+public class EvaluationEvent {
+
+    private final Feature feature;
+
+    private String user = "";
```

Comment:
Do we need to use `String` instead of `string`?

Same question for the `Boolean` below.

Makes sense if these are nullable, but if we're placing defaults like this- I think the lower case version is better.

Reply:
There is no `string` primitive in java. Also, I have a bad habit of always using `B` boolean. FYI, the difference between `Boolean` and `boolean` is that `Boolean` can be `null`.

Reply:
Ah my bad, out of Java practice. I was thinking about about boolean vs Boolean and int vs Integer and incorrectly shortcutted to that existing for String.